### PR TITLE
doc: release notes Feb 2023

### LINKED ISSUE #1412: Release Notes February 2023 release

### PR DESCRIPTION
**Pull request by jsilvela**
Closes #1412 

Signed-off-by: Jaime Silvela <jaime.silvela@enterprisedb.com>

### CODE REVIEW

File: docs/src/release_notes/v1.19.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  exporter, including values equivalent to the `min_wal_size`, `max_wal_size`,
+  `keep_wal_size`, `wal_keep_segments`, as well as the maximum number of WALs
+  that can be stored in the dedicated volume (#1382)
+- Add a database comment on the `streaming_replica` user (#1349)
+- Document webhooks' issues on GKE (#1364)
+- Add note about postgresql.conf in `recovery` (#1211)
+- Add instructions on installing plugin using packages (#1357)
+- Specify Postgres versions supported by each minor release (#1355)
+- Define a PVC group (#1344)
+- Add instructions on installing plugin using packages (#1357)
```
Comment (phisco) [inline docs/src/release_notes/v1.19.md:45]: Duplicated line

File: docs/src/release_notes.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 History of user-visible changes for CloudNativePG, classified for each minor release.
 
+- [CloudNativePG 1.19](release_notes/v1.18.md)
```
Comment (sxd) [inline docs/src/release_notes.md:5]: this shouldn't be `release_notes/v1.19.md` ?

Comment (jsilvela) [reply docs/src/release_notes.md:5]: thank you, sir

File: docs/src/release_notes/v1.19.md (hunk 1 above)
Comment (jsilvela) [reply docs/src/release_notes/v1.19.md]: d'oh. Thanks!

File: docs/src/supported_releases.md [hunk 3]
```diff
@@ -65,8 +65,9 @@ Git tags for versions are prepended with `v`.
 
 | Version         | Currently Supported  | Release Date      | End of Life              | Supported Kubernetes Versions | Tested, but not supported | Supported Postgres Versions |
 |-----------------|----------------------|-------------------|--------------------------|-------------------------------|---------------------------|-----------------------------|
+| 1.19.x          | Yes                  | February 9, 2023  | ~ June 15, 2023          | 1.23, 1.24, 1.25, 1.26        | 1.22                      | 11 - 15                     |
```
Comment (litaocdl) [inline docs/src/supported_releases.md:68]: is EoL should be some day like July 10  for 1.19.x ? totally 5 month duration.

**Review by sxd — APPROVED** [review]

**Review by gbartolini — APPROVED** [review]

**Pull request merged**